setup.py

```diff
@@ -103,7 +101,7 @@
     'Source': 'https://github.com/secure-systems-lab/securesystemslib',
     'Issues': 'https://github.com/secure-systems-lab/securesystemslib/issues',
   },
-  python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4",
+  python_requires="!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4",
```

Comment:
```suggestion
  python_requires="~=3.6",
```
I think the "compatible release" clause that TUF uses works here too (that should translate to any 3.X version where X>=6)

Reply:
Yes, thank you. I even introduced that clause in tuf 🤦